frontend/tests/app/droplets.test.tsx

```
@@ -1,14 +1,13 @@
-//import { render, screen, fireEvent, waitFor } from "@testing-library/react";
-//import Error from "@/app/(droplets)/d/[slug]/[lessonSlug]/error";
-//import { Confetti } from "@/app/(droplets)/d/[slug]/recap/confetti";
-//import NotFound from "@/app/(droplets)/d/[slug]/not-found";
-//import confetti from "canvas-confetti";
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import Error from "@/app/(droplets)/d/[slug]/[lessonSlug]/error";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)